### DISCUSSION TITLE
[BUG] Time and battery, reception icons aren’t visible when using Light Mode in iOS.

### DISCUSSION THREAD

**Discussion by Shohan-Sarker** [aeharding/voyager#164] [General]
![image](https://github.com/aeharding/wefwef/assets/110914258/481b6e07-3505-4afc-9951-f31063caaf43)

It seems to be because of the opening animation being dark, causes the UI to switch to Light icons but then iOS doesn't adapt to the white background? 

Getting this after the newest update.

**Comment by Shohan-Sarker** [comment]
Also (cause i forgot to add in the post) this is the best lemmy app by far! 😃

**Reply by aeharding** [reply]
Unfortunately this is an iOS limitation of progressive webapps. I wish it wasn't.

**Reply by Shohan-Sarker** [reply]
Got it. Sucks that it's a iOS limitation.

If i may, could i suggest for two features?

1. Ability to turn off the display community or user instance.
2. A dark-grey (instead of current pitch black) dark mode toggle. 

a. I (personally) prefer grey opposed to pitch black as it feels easier on the eyes.

b. Grey dark mode is better accessibility wise. It's also recommended to use grey in [material design docs](https://m2.material.io/design/color/dark-theme.html#properties).

**Comment by aeharding** [comment]
Fixed in iOS 17. Or you can use the native app :)
